skyportal/handlers/api/plan.py

```diff
@@ -0,0 +1,450 @@
+import datetime
+import os
+import copy
+
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.orm import joinedload
+from astropy import time
+import ephem
+import gwemopt.utils
```

Comment:
These all need to be imported explicitly? You can't just `import gwemopt` once and then access all of these from the top-level import?

Reply:
Good call!

Reply:
Ok hmmm anything but specifically calling gwemopt.utils complains it doesn't know how to get gwemopt.utils.blah... I guess I can try to fix that in gwemopt packaging. I guess there is a standard way to do that?